src/integration-test/java/com/commercetools/sync/integration/commons/utils/ProductITUtils.java

```diff
@@ -229,5 +235,38 @@ public static Type createPricesCustomType(
         typeKey, locale, name, ResourceTypeIdsSetBuilder.of().addPrices(), ctpClient);
   }
 
+  /**
+   * Builds a {@link PriceDraft} element
+   *
+   * @param amount the amount to create a price draft.
+   * @param currencyUnits the currency unit of the amount.
+   * @param customerGroupId the customer Group Id to create the reference.
+   * @param validFrom
+   * @param validUntil
```

Comment:
it's better to add field description for consistency.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/734/commits/1312aa6580d61010b41f8bd1f00f6202cece6c97